Formula/scrcpy.rb

```diff
@@ -17,10 +12,15 @@ class Scrcpy < Formula
   depends_on "sdl2"
 
   resource "prebuilt-server" do
-    url "https://github.com/Genymobile/scrcpy/releases/download/v1.8/scrcpy-server-v1.8.jar"
-    sha256 "839055ef905903bf98ead1b9b8a127fe402b39ad657a81f9a914b2dbcb2ce5c0"
+    url "https://github.com/Genymobile/scrcpy/releases/download/v1.9/scrcpy-server-v1.9.jar"
+    sha256 "ad7e539f100e48259b646f26982bc63e0a60a81ac87ae135e242855bef69bd1a"
   end
 
+  # include the commit that fixes non portable build
+  patch do
+    url "https://github.com/Genymobile/scrcpy/commit/bcd0a876f7bf52642330410812a6b7100cdeda91.patch?full_index=1"
+    sha256 "a168993bac34e6c3ce5ed3470d55de417c0b881563b3381c9f04f29bdce1a64b"
```

Comment:
Since you added `?full_index=1`, the content of the patch has changed. The new SHA-256 is `86444cd3e26b73cf1c48e442c7aeabc748140b3422c860ac7ebbb05a374c1405`.